### DISCUSSION TITLE
Allow us to click on a user on the /statuses/active map

### DISCUSSION THREAD

**Discussion by thibaultmol** [Traewelling/traewelling#2548] [Ideas / Feature Requests]
It would be nice if you see someone on the map, you could click on them and see the trip they're on

**Comment by pascalwittler** [comment]
This should be easy to do in general by putting `<a>` HTML tags around the already existing `<img>` tags you can already target with the mouse.

In my experience, one problem will be to prevent touch gestures for dragging and zooming the map from colliding with clicks on the linked profile icons.
